README.md

```diff
@@ -6,7 +6,18 @@ Python3 package to cluster VCF records.
 Used by gramtools and minos - written specifically for those projects and no others.
 
 ## Install
-    pip3 install wheel cluster_vcf_records
+
+### pypi
+
+```sh
+pip install wheel cluster_vcf_records
```

Comment:
why pip and not pip3? I bet there are a ton of users where pip != pip3

Reply:
For a few reasons. python2 is "dead", this package won't install on python2, quite a few people use `pyenv` now so `pip` is for the python in their environment. But anyway I'll change it back if it's a problem.